lib/driver.js

```diff
@@ -473,10 +473,7 @@ class AndroidDriver extends BaseDriver {
   }
 
   getPrefsBuilder () {
```

Comment:
i think we can just delete this getPrefsBuilder method now that it doesn't do anything